src/3box.js

```diff
@@ -25,7 +25,8 @@ const config = require('./config.js')
 const API = require('./api')
 
 const ACCOUNT_TYPES = {
-  ethereum: 'ethereum'
+  ethereum: 'ethereum',
```

Comment:
Do we need this type still?

Reply:
technically if anyone passed it in isAccountLinked, but don't thing it would have been used at all there, could be removed